packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -107,6 +107,7 @@ const TSErrors = makeErrorTemplates(
     InvalidModifierOnTypeMember:
       "'%0' modifier cannot appear on a type member.",
     InvalidModifiersOrder: "'%0' modifier must precede '%1' modifier.",
+    InvalidTuple: "Invalid Tuple.",
```

Comment:
We can have a better error, like `InvalidTupleElementKeyword: "The '%0' keyword cannot be used as a tuple element."`

Reply:
done in https://github.com/babel/babel/pull/13743/commits/93be0ff35165a828a71f8c30ee18d8fe3cec6989